pyvcloud/vcd/vdc.py

```diff
@@ -509,6 +509,24 @@ def get_disk(self, name, disk_id=None):
                         return disk
         return None
 
+    def change_disk_owner(self, name, user_href, disk_id=None):
+
+        if self.resource is None:
+            self.resource = self.client.get_resource(self.href)
+
+        if disk_id is not None:
+            disk = self.get_disk(None, disk_id)
+        else:
+            disk = self.get_disk(name)
+        new_owner = disk.Owner
```

Comment:
This works, but I think a better way would be to strip off the entire User section of the disk element and add back a new User element which has a single attribute viz. href.

Reply:
User section has only two fields viz., name and href. I prefer to keep as-is this time as it doesn't make much difference. I will keep this in mind for future, thanks.